scripts/finschia/start.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck >/dev/null && shellcheck "$0"
+
+# Please keep this in sync with the Ports overview in HACKING.md
+TENDERMINT_PORT_GUEST="26658"
+TENDERMINT_PORT_HOST="26658"
+API_PORT_GUEST="1317"
+API_PORT_HOST="1317"
+GRPC_PORT_GUEST="9090"
+GRPC_PORT_HOST="9090"
+
+# If this error `realpath: command not found` occurs, run the following command.
+# brew install coreutils
+SCRIPT_DIR="$(realpath "$(dirname "$0")")"
+source "$SCRIPT_DIR"/env
+TMP_DIR=$(mktemp -d "${TMPDIR:-/tmp}/fnsa.XXXXXXXXX")
+chmod 755 "$TMP_DIR"
+echo "Using temporary dir $TMP_DIR"
```

Comment:
how about ">&2" if it is needed?

Reply:
Fixed here.
https://github.com/Finschia/finschia-wasm/pull/3/commits/f5c03a42042926bd99ff287f22c0c344a3c50256